src/GraphQL/Execution/IExecutionContext.cs

```diff
@@ -72,7 +73,7 @@ public interface IExecutionContext : IProvideUserContext
         /// A delegate that can override, hide, modify, or log unhandled exceptions before they are stored
         /// within <see cref="Errors"/> as an <see cref="ExecutionError"/>.
         /// </summary>
-        Action<UnhandledExceptionContext> UnhandledExceptionDelegate { get; }
+        Func<UnhandledExceptionContext, Task> UnhandledExceptionDelegate { get; set; }
```

Comment:
```suggestion
        Func<UnhandledExceptionContext, Task> UnhandledExceptionDelegate { get; }
```

Reply:
Everything else in `IExecutionContext` is readonly.

Reply:
Of course, just copy-paste.